### PR TITLE
Sprint-Challenge-Redux-Smurfs

### PR DESCRIPTION
**Pull request by vijaydas**

### CODE REVIEW

File: smurfs/src/components/Smurf.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    //   }
+
+render() {
+    return (
+    <div className="SmurfBox">
+    <h3>{this.props.smurf.name}</h3>
+     <strong>{this.props.smurf.height} tall</strong>
+     <p>{this.props.smurf.age} smurf years old</p>
+    )}
+     </div>
```
Comment (lennamau) [inline smurfs/src/components/Smurf.js:19]: Remove extra characters here )} they are showing up on your list.

File: smurfs/src/components/SmurfList.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        this.props.getSmurfs();
+    }
+
+    render() {
+        return (
+            <div className="Smurfs">
+                {this.props.smurfs.map(smurf => {
+                    return <Smurf smurf={smurf} />
+                })}>
+
```
Comment (lennamau) [inline smurfs/src/components/SmurfList.js:17]: Extra character (<) here is showing up on when you add a smurf. Need to remove.

**Comment by lennamau** [comment]
Sprint Redux-Smurfs
## MVP
- [x] Thunk
- [x] GET Smurfs
- [x] POST Smurfs
- [x] Actions
- [x] Reducers
- [x] wire everything up inside of your component tree

## Stretch
- [ ] PUT request
- [ ] DELETE 

## What Looks Great
- Great work completing MVP
- Code is well formatted and easy to read
- Good job with frequent commits and descriptive messages


## What Could Be Improved
- Route back to list after adding
- Try some stretch goals
- Remove extra characters as noted in code

## Rating (1-3)
### 2

File: smurfs/src/components/Smurf.js (hunk 1 above)
Comment (vijaydas) [reply smurfs/src/components/Smurf.js:19]: Fixed and committed.

File: smurfs/src/components/SmurfList.js (hunk 2 above)
Comment (vijaydas) [reply smurfs/src/components/SmurfList.js:17]: Fixed and committed.

**Pull request merged**